example/core_v1alpha1_observatorium.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: core.observatorium.io/v1alpha1
+kind: Observatorium
+metadata:
+  name: observatorium-xyz
+  app.kubernetes.io/component: observatorium-cr
+  app.kubernetes.io/instance: observatorium-xyz
+  app.kubernetes.io/name: observatorium-cr
+  app.kubernetes.io/part-of: observatorium
+spec:
+  thanosImage: "quay.io/thanos/thanos:"
```

Comment:
hm this is kind of weird, no? The images really should not end in a `:`

Reply:
See here:
https://github.com/observatorium/configuration/blob/master/environments/base/default-config.libsonnet#L7

Reply:
sure, but this is still incorrect. why isn't this `thanosImage: "quay.io/thanos/thanos:master-2020-02-13-adfef4b5`

Reply:
Done